src/lib/helpers/faker.ts

```
@@ -192,6 +198,25 @@
+            return [faker.location.longitude(), faker.location.latitude()];
+        }
+
+        case 'linestring': {
+            return Array.from({ length: 5 }, () => [
+                faker.location.longitude(),
+                faker.location.latitude()
+            ]);
+        }
+
+        case 'polygon': {
+            let points = Array.from({ length: 5 }, () => [
```

ESLint

[error] 'points' is never reassigned. Use 'const' instead.

(prefer-const)